docs/analyzer-rules/AZFW0009.md

```diff
@@ -0,0 +1,28 @@
+# AZFW0008: Invalid Retry Options
```

Comment:
AZFW009 10 and 11 are coming in from sdkbindings branch - so maybe use 12 here? 

Also mismatch in file name number and heading number

Reply:
Also 8 and 9 already exist in main: https://github.com/Azure/azure-functions-dotnet-worker/tree/aa79ebedc9c775532d5f14903a96d835fef05707/docs/analyzer-rules

Reply:
Ok let me skip to 12